File: tuts/012-transitgateway-gettingstarted/php/GettingStartedScenario.php
<?php
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

require 'vendor/autoload.php';

use Aws\Ec2\Ec2Client;

$client = new Ec2Client(['region' => 'us-east-1']);
echo "Running Ec2 getting started scenario...\n";
// TODO: setup, interact, teardown
echo "Scenario complete.\n";

File: tuts/003-s3-gettingstarted/swift/Sources/GettingStartedScenario.swift
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import AWSS3

@main
struct GettingStartedScenario {
    static func main() async throws {
        let client = try S3Client(region: "us-east-1")
        let wrapper = S3Wrapper(client: client)
        print("Running S3 getting started scenario...")
        // TODO: setup, interact, teardown
        let _ = wrapper
    }
}
